src/App.tsx

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const TODO_DEFAULT: Todo = {
+    id: 0,
+    title: '',
+    completed: false,
+    userId: 0,
+  };
```

Comment:
Everything that can be moved outside of component should be moved outside of component.